### DISCUSSION TITLE
EDDI personality

### DISCUSSION THREAD

**Discussion by CMDR-Evolution** [EDCD/EDDI#2267] [Share Personalities]
Hello EDDI community,

I'm a massive fan of EDDI and have played Elite for 6,534 hours (as of Oct 1st 2021). 

From spending so many hours on Elite using EDDI, I've made various adjustments over the years to mostly give EDDI more ways to say information and other small changes. I'm not a developer so it's not drastically different and won't be alien to you, but you'll hopefully find it to be an enhancement.

**Some things to note:**

- This will be kept updated with any changes.
- Please give me a shout if you have any questions or issues and I'll be happy to investigate.
- As with any TTS, certain words are pronounced slightly differently. I personally use IVONA 2 Amy - speed 0 - processing 30.
- You only need to adjust the 'honorific' script to use your name instead as it's referenced in a lot of places.
- If you have some non-English / unique ship names [(like my fleet here)](https://inara.cz/cmdr-fleet/80559/), you can add the correct pronunciations to the 'ships' script. If you don't want to add anything there, the standard name will be used so it'll work either way ;).
- I'm not able to upload the JSON file, so you'll find the full personality in a plain text file attached. 
- Just download the file and adjust it to a json file format, then it will work o7.

[elite.txt](https://github.com/EDCD/EDDI/files/7774282/elite.txt)

**Comment by DaVidoSS0** [comment]
Hi. I just read your biography and OMG. It is how it's meant to be played. It was a pleasure, thank you.

**Reply by CMDR-Evolution** [reply]
Hey DaVidoSSO, thank you very much! Glad you enjoyed having a look through. 

While I'm here I have to apologise as I've been meaning to upload a much more improved version of the EDDI personality for quite a while. 

Trouble is, I keep thinking of improvements, and then it's never finished lol, a lot like Elite o7. 

Update 30/10/25. I'll have a new version ready and shared around 20/11/25

**Comment by VladMVLX** [comment]
Hi @CMDR-Evolution 
thanks for the share it is very lovely personality, do you have any updates on it? 
It is having error in star report script every time you visit new system. Also, errors on volcanic and atmospheric reports when scanning the planet.

**Reply by CMDR-Evolution** [reply]
Hello Vlad. Sorry for the very slow reply. I had a break from Elite to move house and had a lot of work to do. I've been back since May and have been constantly working on EDDI as well. I'll have a much more enhanced version to share around 20/11 o7.

**Comment by nepomuk16321** [comment]
Hi @VladMVLX , I don't know @CMDR-Evolution 's character as he is too old for the current EDDI. But I can recommend the character of CMDR Darkcyde. I also use parts of his character. You can find his character by logging into the [FD forum](https://forums.frontier.co.uk/threads/eddi-scripts-and-eddi-enabled-va-commands-thread.311614/post-10477830) and looking at a post by CMDR Darkcyde (attached at the bottom).

**Reply by VladMVLX** [reply]
@nepomuk16321 awesome thank you a lot!

**Reply by nepomuk16321** [reply]
You're welcome.

**Reply by CMDR-Evolution** [reply]
Thanks NepomUK. Sorry for not getting back to you. Yes, An upgrade to EDDI broke the personality while I was having a break from Elite due to moving house and needing to do a lot of work on it. I've been back since May and have been spending many, many hours on EDDI. It's fully working and has countless upgrades from my last upload. I'm working on the last tweaks and will share it in a week or two.

**Reply by nepomuk16321** [reply]
Hello CMDR Evolution, it's great to have you back. ELITE never lets you go, does it? I hope everything is okay with your home and family so that you can spend a little more time with ELITE again. Oh, and I thought your greeting to me was really funny, “NepomUK.” I'll remember that for future accounts. I'm not from the UK at all, but from Germany. The name Nepomuk comes from a saint. He is known as the patron saint of bridges in Czechia and Germany.
https://en.wikipedia.org/wiki/John_of_Nepomuk

**Reply by CMDR-Evolution** [reply]
Ah, my apologies Nepomuk, and also my thanks for teaching me something new! Apologies, I didn't get a notification for your message, but just randomly came back here and saw it. 

For re-sharing my template, I've been putting a lot of work in, and although I said I'd share it in two weeks, I have an AI subscription until 20/11, so I'm thinking I should press on while that's active. Otherwise, I'd be sharing it and then again a couple of weeks later. I'll carry on working on it most of my evenings and weekends to make it as refined as possible o7
